app/about-us.html

```diff
@@ -219,7 +219,8 @@ <h1>Sound Like <span class = "underline">you?</span></h1>
           <a href="https://clutch.co/profile/therefore-interactive" target="_blank"><img src="images/web_developers_canada_2017.png" alt="Clutch Rated Top Development Design Agency"></a>
         </div>
       </div>
-      <div class="copyright">© 2018 Therefore Interactive Inc. All Rights Reserved.</div>
+      <div class="copyright">© <span class ="copyright-year">2019</span> Therefore Interactive Inc. All Rights Reserved.</div>
```

Comment:
@chandantherefore Is there something to update `.copyright-year` in js to keep the value dynamic?

Reply:
yes, i used it in main.js to make it dynamic.